x-pack/plugins/apm/server/plugin.ts

```diff
@@ -126,7 +125,7 @@ export class APMPlugin
     const getCoreStart = () =>
       core.getStartServices().then(([coreStart]) => coreStart);
 
-    const ready = once(async () => {
+    const initializeRuleDataIndices = once(async () => {
```

Comment:
It doesn't really initialize the rule data indices - just the assets needed for index creation (that part happens when writing with `RuleDataWriter.bulk`).

Reply:
true, I'll try to find a better name